obspy/io/sac/tests/test_core.py

```diff
@@ -874,6 +874,16 @@ def test_not_ascii(self):
         self.assertEqual(tr.stats.channel, 'HHE')
         self.assertEqual(tr.stats.network, '')
 
+    def test_sac_booleans_from_trace(self):
+        """
+        SAC booleans `lcalda` and `lpspol` should be `False` and `True`,
+        respectively, by default when converting from a `Trace`.
```

Comment:
Thank you!  I just don't want to learn ReST, apparently :stuck_out_tongue_winking_eye: .